status-im/status-react
# Transaction sheet preview (bounty)

Fixes #16264 

### Summary
Implemented preview for transaction sheet
<img src="https://github.com/status-im/status-mobile/assets/15213331/4f07e7e0-1eb5-4296-894c-e57e577e9c2e" width="400">
<img src="https://github.com/status-im/status-mobile/assets/15213331/67dba793-da7a-445b-b754-d8e21f329e54" width="400">

1. Preview screen for transaction sheet (partial)
2. Locked input component (used in preview screen twice)

#### Fixed:
1. Account selector component to match the design
2. purple 50 colors (to also match the design)

On a previous screen I've re-used components like:
1. account-selector
2. segmented-control

### Steps to test


- Open Status
- Unlock app
- Press the top-left user settings button
- In Settings, press Quo2.0 Preview
- Scroll to "transaction-sheet"
- Slide the account selectors list horizontally
- Ensure that all components are in place
